Clarify what file client config configurations are added to (#818)
micronaut-projects/micronaut-kubernetes: src/main/docs/guide/config-client.adoc

The configuration client by default will read all the ``ConfigMap``s for the configured namespace. You can further filter
which config map names are processed by defining `kubernetes.client.config-maps.includes` or

> `kubernetes.client.config-maps.excludes` in `bootstrap.yml`:

Reviewer: ```suggestion
`kubernetes.client.config-maps.excludes` in https://docs.micronaut.io/latest/guide/#bootstrap[bootstrap configuration]:
```

> [source,yaml]

Reviewer: ```suggestion
[configuration]
```

While you could potentially use the `java.io` or `java.nio` APIs to read the contents yourself, this configuration module
can convert them into a ``PropertySource`` so that you can consume the values much more easily. In order to do so, define

> the following configuration in `bootstrap.yml`:

Reviewer: ```suggestion
the following configuration in https://docs.micronaut.io/latest/guide/#bootstrap[bootstrap configuration]:
```

> [source,yaml]

Reviewer: ```suggestion
[configuration]
```

----

The configuration client, by default, will read all the ``Secret``s for the configured namespace. You can further filter

> which config map names are processed by defining `kubernetes.client.secrets.includes` or `kubernetes.client.secrets.excludes` in `bootstrap.yml`:

Reviewer: ```suggestion
which config map names are processed by defining `kubernetes.client.secrets.includes` or `kubernetes.client.secrets.excludes` in https://docs.micronaut.io/latest/guide/#bootstrap[bootstrap configuration]:

```

> [source,yaml]

Reviewer: ```suggestion
[configuration]
```
